packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 50.5.0 - November 11, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-core
```

Comment:
I think it would be useful to add that the new min version of VSCode we support is 1.46.0 https://github.com/forcedotcom/salesforcedx-vscode/pull/2719